snowplow-common-internal/kinesis/src/it/scala/com/snowplowanalytics/snowplow/sources/Utils.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Snowplow Community License Version 1.0,
+ * and you may not use this file except in compliance with the Snowplow Community License Version 1.0.
+ * You may obtain a copy of the Snowplow Community License Version 1.0 at https://docs.snowplow.io/community-license-1.0
+ */
+package com.snowplowanalytics.snowplow.sources
+
+import cats.effect.{IO, Ref}
+
+import org.testcontainers.containers.localstack.LocalStackContainer
+import org.testcontainers.containers.localstack.LocalStackContainer.Service
+import org.testcontainers.containers.wait.strategy.Wait
+import org.testcontainers.utility.DockerImageName
+
+import software.amazon.awssdk.core.SdkBytes
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.kinesis.KinesisAsyncClient
+import software.amazon.awssdk.services.kinesis.model.{PutRecordRequest, PutRecordResponse}
+
+import java.net.URI
+import java.nio.charset.StandardCharsets
+import java.util.UUID
+
+object Utils {
+
+  def testProcessor(ref: Ref[IO, List[String]]): EventProcessor[IO] =
+    _.evalMap { case TokenedEvents(events, token) =>
+      for {
+        _ <- ref.update(_ ::: events.map(bytes => new String(bytes, StandardCharsets.UTF_8)))
+      } yield token
+    }
+
+  def getLocalstackContainer(
+    exposedPort: Int,
```

Comment:
What about defining it in `LOCALSTACK_EXPOSED_PORT` in this file and using it as default ? If we have several tests, we still want only one localstack instance with the same exposed port.